server/src/main/java/com/senlainc/warsaw/tyurin/dao/CraftsmanDAO.java

```diff
@@ -6,16 +6,20 @@
 import com.senlainc.warsaw.tyurin.entity.Craftsman;
 import com.senlainc.warsaw.tyurin.util.EntityBuilder;
 import com.senlainc.warsaw.tyurin.util.dbConnection.DBConnector;
+import org.apache.log4j.Logger;
 
 import java.sql.Connection;
 import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
 
 @DependencyClass
 public class CraftsmanDAO implements ICraftsmanDAO{
```

Comment:
> Class names should be nouns, in mixed case with the first letter of each internal word capitalized. Try to keep your class names simple and descriptive. Use whole words-avoid acronyms and abbreviations (unless the abbreviation is much more widely used than the long form, such as URL or HTML).

Usually in production development reviewers asks to change to Dao, it's a matter of taste but still a thing so
-> CraftsmanDao